<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uartTest</class>
 <widget class="QWidget" name="uartTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uartTest</string>
  </property>
  <widget class="QGroupBox" name="ALL">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>480</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="PORT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>PORT SELECT</string>
    </property>
    <widget class="QComboBox" name="PORT_BOX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>COM 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 15</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="BAUDRATE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>BAUDRATE</string>
    </property>
    <widget class="QComboBox" name="BAUD_BOX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="Title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UART TEST</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="INPUT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#groupBox_4{


	font: 75 16pt &quot;Agency FB&quot;;


}</string>
    </property>
    <property name="title">
     <string>INPUT</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Input_bt">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>INPUT</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>OUTPUT</string>
    </property>
    <widget class="QListWidget" name="OUTPUT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exit_bt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
